config/manager/manager.yaml

```diff
@@ -55,7 +55,7 @@ spec:
         operator: Exists
       containers:
       - name: ibmcloud-machine-controller
-        image: quay.io/cluster-api-provider-ibmcloud/controller:v0.0.1
+        image: quay.io/cluster-api-provider-ibmcloud/controller:master
```

Comment:
as a version, `latest` seems better than `master`

Reply:
+1 to use latest.

Reply:
Done